xmipptomo/protocols/protocol_phantom_tomo.py

```diff
@@ -107,22 +112,17 @@ def createPhantomsStep(self):
         minDim = min(xT, yT, zT)
         self.info("Minimum dimension: %s" % minDim)
 
-        radius = max(4, int(minDim * 0.05))
-        height = max(10, int(minDim * 0.1))
-        self.info("Cone radius, height :%s, %s" % (radius, height))
+        boxSize = max(30, int(minDim * 0.1))
```

Comment:
I think 32 instead of 30 is a most common dimension (but this is not so important)